aries-site/src/examples/templates/wizard/TwoColumnWizardExample.js

```diff
@@ -175,7 +176,7 @@ const steps = [
   },
 ];
 
-export const TwoColumnWizardExample = () => {
+export const TwoColumnWizardExample = ({ containerRef }) => {
```

Comment:
Can we copy and paste the comment from the Filtering file about why this prop is here?

Reply:
yes I will add